tests/test_factory.py

```diff
@@ -10,7 +11,7 @@
 
 
 @pytest.mark.usefixtures("arrow_factory")
-class TestGet:
+class TestGet(unittest.TestCase):
```

Comment:
No need to use `unittest` here. We are using `pytest`, so all you need to do to test this is:

```python
with pytest.raises(ValueError):
    self.factory.get(None)
```

Reply:
How silly of me the example was right below, it has been changed.